inference-engine/thirdparty/clDNN/CMakeLists.txt

```diff
@@ -95,3 +105,7 @@ if(CLDNN__INCLUDE_TESTS)
 endif()
 
 add_subdirectory(kernel_selector)
+
+if(CLDNN_THREADING MATCHES "TBB")
+  target_link_libraries(${CLDNN_BUILD__PROJ__clDNN} INTERFACE inference_engine_s)
```

Comment:
I expect it can result in binary size increase for clDNNPlugin because it will link a lot of IE methods statically.

Reply:
yes we are trying to switch this to shared library (libinferrence_engine.so) but it is failing to find that lib. Could you give us any hint if you know why?

Reply:
It was due to the symbol visibility problem: custom task_arena's symbols are not included in the shared lib. 
There is no trivial solution for now, we will close this for now and revisit later with better alternative.